.github/workflows/koupleless_adapter_release.yml

```
@@ -0,0 +1,79 @@
+          distribution: 'temurin'
+          cache: maven
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+
+      - name: Finding Maven Modules
+        id: find-maven-modules
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:24: Double quote to prevent globbing and word splitting

(shellcheck)